arangod/Metrics/MetricsFeature.cpp

```diff
@@ -55,308 +55,459 @@ MetricsFeature::MetricsFeature(application_features::ApplicationServer& server)
   setOptional(false);
   startsAfter<LoggerFeature>();
   startsBefore<application_features::GreetingsFeaturePhase>();
+  nameVersionTable = {
```

Comment:
@MBkkt I suggest we use `frozen::unordered_map` instead of `std::unordered_map`. I'd also move `nameVersionTable` into `cpp` completely

Reply:
Done